committee-code-of-conduct/README.md

```diff
@@ -3,12 +3,16 @@
 The Kubernetes Code of Conduct Committee (CoCC) is the body that is responsible for enforcing and maintaining the Kubernetes Code of Conduct.
 
 The members and their terms are as follows:
-- Jaice Singer Dumars (Google) - 2 years
-- Paris Pittman (Google) - 2 years
+
+###Term ends on August 7, 2019
```

Comment:
This is missing a space between the `###` and the heading so it's not rendering right. Same for the one below

Reply:
thank you!